docs/modules/ROOT/pages/user-guide/testing.adoc

```diff
@@ -226,3 +226,20 @@ class MyTest {
 ----
 
 More examples of WireMock usage can be found in the Camel Quarkus integration test source tree such as https://github.com/apache/camel-quarkus/tree/main/integration-tests/geocoder[Geocoder].
+
+== Testing with `CamelTestSupport` in JVM mode
```

Comment:
@ppalaga I added a chapter about jvm testing. Fro my PoV I'd like to add a list of features, which are supported (and covered by junit tests)